work/v1/types.go

```diff
@@ -170,7 +170,7 @@ type ManifestCondition struct {
 type ManifestConditionType string
 
 const (
-	// ManifestProgressing represents the resource is being applied on the managed cluster
+	// ManifestProgressing represents the resource the is being applied on the managed cluster
```

Comment:
This was my error. This should be "that", not "the".

Reply:
fixed